tests/ArmTemplates.Tests/Extractor/Scenarios/ApiExtractorTests.cs

```diff
@@ -227,11 +227,76 @@ public async Task GenerateGraphQLApiTemplates()
             string schemaContentType = "application/vnd.ms-azure-apim.graphql.schema";
 
             // api schemas
-            apiTemplate.TypedResources.ApiSchemas.Count().Should().Be(2);
+            apiTemplate.TypedResources.ApiSchemas.Count().Should().Be(3);
             apiTemplate.TypedResources.ApiSchemas.All(x => x.Type == ResourceTypeConstants.APISchema).Should().BeTrue();
             apiTemplate.TypedResources.ApiSchemas.All(x => x.Properties is not null).Should().BeTrue();
             apiTemplate.TypedResources.ApiSchemas.All(x => x.Properties.Document.Value.ToString().Equals(fileReadingTask.Result.ToString())).Should().BeTrue();
             apiTemplate.TypedResources.ApiSchemas.All(x => x.Properties.ContentType.Equals(schemaContentType)).Should().BeTrue();
         }
+
+        [Fact]
+        public async Task GenerateApiTemplateAsync_WebsocketApiTypeOperationsNotGenerated()
```

Comment:
can we do test, where there are both websocket and not websocket apis?

Reply:
This method's mocked clients include WebSocket Api types as well as http.
Mocked Api client returns 3 apis with http and websocket types